src/aleph_client/commands/utils.py

```diff
@@ -200,3 +206,111 @@ def is_environment_interactive() -> bool:
+    table.add_column("Reward Address", style="green", justify="center")
+    table.add_column("Address", style="green", justify="center")
+
+    console = Console()
+    progress = Progress()
+    task = progress.add_task("[green]Fetching node info... It might take some time", total=len(node_info.nodes))
+
+    progress_table = ProgressTable(progress, table)
+    item_hashes = []
+
+    queue = asyncio.Queue()
+
+    async with aiohttp.ClientSession() as session:
+        with Live(progress_table, console=console, refresh_per_second=2):
+            fetch_task = asyncio.create_task(fetch_data(session, node_info, queue, progress, task, item_hashes))
+            update_task = asyncio.create_task(update_table(queue, table))
+            await asyncio.gather(fetch_task, update_task)
+
+    return item_hashes
+
+async def fetch_data(session: aiohttp.ClientSession, node_info: NodeInfo, queue: asyncio.Queue, progress: ProgressTable, task: list, item_hashes: list):
+    tasks = [fetch_and_queue(session, node, queue, progress, task, item_hashes) for node in node_info.nodes]
+    await asyncio.gather(*tasks)
+    await queue.put(None)
+
+async def fetch_and_queue(session: aiohttp.ClientSession, node: NodeInfo, queue: asyncio.Queue, progress: Progress, task: list, item_hashes: list):
+    try:
+        system_info, version = await asyncio.gather(
+            fetch_crn_system(session, node),
+            get_crn_version(session, node)
+        )
+        async with session.get(node["address"] + "status/check/ipv6", timeout=10) as resp:
+            if resp.status == 200:
+                node_stream = node["stream_reward"]
+                if node_stream and system_info:
+                    node_name = _escape_and_normalize(node["name"])
+                    node_name = _remove_ansi_escape(node_name)
+                    node_address = node["address"]
+                    score = _format_score(node["score"])
+                    cpu = f"{system_info['cpu']['count']} {system_info['properties']['cpu']['architecture']}"
+                    hdd = f"{system_info['disk']['available_kB'] / 1024 / 1024:.2f} GB"
+                    ram = f"{system_info['mem']['available_kB'] / 1024 / 1024:.2f} GB"
+                    await queue.put((score, node_name, cpu, ram, hdd, version, node_stream, node_address))
+                    item_hashes.append(node_stream)
+    except Exception as e:
+        
+        pass
+    finally:
+        progress.update(task, advance=1)
+
+async def update_table(queue: asyncio.Queue, table: Table):
+    while True:
+        data = await queue.get()
+        if data is None:
+            break
+        table.add_row(*data)
+
+async def fetch_crn_system(session: aiohttp.ClientSession, node: NodeInfo):
+    try:
+        async with session.get(node["address"] + "about/usage/system", timeout=10) as resp:
+            if resp.status == 200:
+                data = await resp.json()
```

Comment:
Would it make sense to use the model `CpuProperties` from aleph-message ?

```suggestion
                data_raw = await resp.json()
                data = CpuProperties.parse_obj(data_raw)
```

See https://github.com/aleph-im/aleph-message/blob/3c0144adb73d54d146cb6ce407a6bf5a14cf7995/aleph_message/models/execution/environment.py#L71